src/database/queries/sessions.js

```diff
@@ -341,7 +360,7 @@ exports.countHostedSessions = async (id) => {
 		const option = {
 			attributes: ['session_id'],
 		}
-		const sessionIds = await sessionOwnership.findAll(filter, option, true)
+		const sessionIds = await sessionOwnership.findAll(filter, tenantCode, option, true)
```

Comment:
@sumanvpacewisdom remove sessionOwnership model and other model which is not require